emukit/examples/multi_fidelity_dgp/benchmarking_examples.ipynb

```diff
@@ -0,0 +1,288 @@
+{
```

Comment:
<p>Nitpick: you could have a little named tuple representing each function. and then instead of `num_data_dict[fct]` you would do `fct.num_data_dict`, which looks a bit nicer</p>

 _Reply via <a href='https://app.reviewnb.com/amzn/emukit/pull/181/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
I'll do this then push